frontend/src/components/common/header/header.tsx

```diff
@@ -75,18 +75,28 @@ const Header: FC = () => {
           )}
           <div className={styles.userWrapper}>
             {hasUser ? (
-              <button onClick={handlePopupOpen} className={styles.button}>
-                <div className={styles.imageWrapper}>
-                  <Image
-                    width="50"
-                    height="50"
-                    src={avatarUrl ?? defaultUserAvatar}
-                    alt="user avatar"
-                    isCircular
-                    className={styles.img}
-                  />
-                </div>
-              </button>
+              <div className={styles.popupWrapper}>
+                <button onClick={handlePopupOpen} className={styles.button}>
+                  <div className={styles.imageWrapper}>
+                    <Image
+                      width="50"
+                      height="50"
+                      src={avatarUrl ?? defaultUserAvatar}
+                      alt="user avatar"
+                      isCircular
+                      className={styles.img}
+                    />
+                  </div>
+                </button>
+                <button onClick={handlePopupOpen} className={styles.button}>
+                  <div className={styles.caretImage}>
+                    <Icon
+                      name={isMenuPopupVisible ? 'caretUp' : 'caretDown'}
+                      className={styles.caretIcon}
+                    ></Icon>
```

Comment:
```suggestion
                    />
```
why not self closing tag?